Enable multiple profile inheritance.

[main]
include=profile1;profile2

Also comma "," can be used instead of the semicolon ";".
No escaping is done, so it's assumed there will be no comma or semicolon
in the profile name.

Signed-off-by: Jaroslav Skarvada <jskarvad@redhat.com>

See: https://github.com/redhat-performance/tuned/commit/feb6e8b6f4fd3c1b1595fc52c8b0868a2460a819

diff --git a/tuned/profiles/loader.py b/tuned/profiles/loader.py
index df7bd9e4..e844472f 100644
--- a/usr/lib/python2.7/site-packages/tuned/profiles/loader.py
+++ b/usr/lib/python2.7/site-packages/tuned/profiles/loader.py
@@ -78,8 +78,8 @@ def _load_profile(self, profile_names, profiles, processed_files):
 			config = self._load_config_data(filename)
 			profile = self._profile_factory.create(name, config)
 			if "include" in profile.options:
-				include_name = self._variables.expand(profile.options.pop("include"))
-				self._load_profile([include_name], profiles, processed_files)
+				include_names = re.split(r"\b\s*[,;]\s*", self._variables.expand(profile.options.pop("include")))
+				self._load_profile(include_names, profiles, processed_files)
 
 			profiles.append(profile)
 
